src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <IsPackable>true</IsPackable>
+    <!-- This library needs more bake time until it can ship stable. -->
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
Just for my own understanding as this is the first time I have seen this one

https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Versioning.md?plain=1#L119-L120